packages/loader/container-loader/src/container.ts

```diff
@@ -359,7 +359,7 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
                         container.removeListener("closed", onClosed);
                     })
                     .then((props) => {
-                        event.end(props);
+                        event.end({ ...props, ...mode });
```

Comment:
Might be worth using ...loadOptions.loadMode instead, to ensure we capture what was actually passed in - we can always look at the code to see what are defaults.